File: 4_swift/esPalindromo.swift
#!/usr/bin/swift

func esPalindromo(palabra: String) -> Bool {
    let invertida = String(palabra.reversed()).lowercased()
    return palabra.lowercased() == invertida
}

print(esPalindromo(palabra: "mex"))
print(esPalindromo(palabra: "Lol"))

File: 3_javascript/esPalindromo.js
function esPalindromo(palabra) {
  let invertida = "";
  for (let i = palabra.length - 1; i >= 0; i--) {
    invertida += palabra[i];
  }
  return palabra.toLowerCase() === invertida.toLowerCase();
}

exports.esPalindromo = esPalindromo;

console.log(esPalindromo("mex"));
console.log(esPalindromo("Lol"));
